tests/data_files/base64/def_bad_b64_eof.txt

```diff
@@ -0,0 +1 @@
+/w==
```

Comment:
There's no “EOFish” character (byte 0xff) here. This is a well-formed file containing Base64 data. The encoded data isn't a valid SSL context.

Reply:
> There's no “EOFish” character (byte 0xff) here. This is a well-formed file containing Base64 data. The encoded data isn't a valid SSL context.

Thanks for your review. 
It is actually base64 encoding of 0xFF.  So, not sure what do you mean by "no "EOFish""

And yes,  it is not a valid SSL Context. It is just meant to represent EOF to test that it can be interpreted correctly by read_next_b64_code() function.

Am I missing something ?

Thanks & Regards,
    - Nayna

Reply:
The bug was in the parsing of the base64-encoded, not in the handling of the decoded data. So the fact that the decoded data contains a 0xff is irrelevant. What matters is when the file itself contains a 0xff.